VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -432,7 +461,12 @@ def controller_id_and_lun_to_settings_data(scsi_controller, lun_number):
         data_disks_settings_data = [controller_id_and_lun_to_settings_data(scsi_controller, lun_number)
                                     for (scsi_controller, lun_number) in data_disk_controller_ids_and_luns]
 
-        data = {"DiskEncryptionDataVersion": self._DISK_ENCRYPTION_DATA_VERSION_V4,
+        if self._DISK_ENCRYPTION_KEY_SOURCE == CommonVariables.KeyStoreTypeManagedHSM:
```

Comment:
Is the Data version change required for disable to?

Reply:
@ShridharParameshwarBhat can you answer this one? I am not sure

Reply:
Sending 4.0